jOOQ-codegen-maven/pom.xml

```diff
@@ -13,6 +13,10 @@
     <name>jOOQ Codegen Maven</name>
     <packaging>maven-plugin</packaging>
 
+    <prerequisites>
+        <maven>3.2.5</maven>
+    </prerequisites>
+
```

Comment:
Not sure why this is needed, won't be included in https://github.com/jOOQ/jOOQ/issues/15065

Reply:
OK, but that's just one more thing to keep an eye out for whether it's up to date or not, right? The Maven plugins probably have higher "meta standards" here, but for jOOQ, this isn't really so important. E.g. we don't have integration test checking if stuff works with various maven versions.